README.md

```
@@ -1,2 +1,32 @@
+|---|---|
+|[`_targets.R`](_targets.R) | The [targets](https://books.ropensci.org/targets/) pipeline used to generate the figures and results in this work. |
+|[`targets`](targets) | The folder containing the files of targets lists grouped by their outputs. |
+|[`input`](input) | Contains the configuration file to specify the evaluation runs. |
+|[`R`](R) | Functions needed to generate targets. |
+|[`docs`](docs) | Files neede to generate the supplement plus a rendered version of the latest Supplement. |
+|[`report.Rmd`](report.Rmd) | Report containing tables with all the summary metrics needed to write the main text results. |
+
+## Data sources
+
+### Raw data used to produce nowcasts generated in this code-base
+The pre-processed reporting triangle used to generate the German COVID-19 nowcasts is available at https://raw.githubusercontent.com/KITmetricslab/hospitalization-nowcast-hub/11c745322c055cfbd4f0c8f72241642a50aea399/data-truth/COVID-19/COVID-19_hospitalizations_preprocessed.csv"
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)